Sphinx v4.1.2 documentation fails

sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v4.1.2

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
File "/usr/lib/python3.10/site-packages/sphinx/config.py", line 327, in eval_config_file
exec(code, namespace)
File "/builddir/build/BUILD/pyspotify-770aee08de274951b63e60bdd8835188b58e7862/docs/conf.py", line 67, in <module>
setattr(cls, method_name, method.__wrapped__)
TypeError: cannot set 'maketrans' attribute of immutable type 'str'

make: *** [Makefile:45: html] Error 2

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -60,7 +60,7 @@ for mod_name, mod in vars(spotify).items():
     if not isinstance(mod, types.ModuleType) or mod_name in ("threading",):
         continue
     for cls in vars(mod).values():
-        if not isinstance(cls, type):
+        if not isinstance(cls, type) or not cls.__module__.startswith("spotify"):
             continue
         for method_name, method in vars(cls).items():
             if hasattr(method, "__wrapped__"):

